go/pserver/client/etcd_client.go

```diff
@@ -0,0 +1,124 @@
+package client
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/PaddlePaddle/Paddle/go/pserver"
+	"github.com/coreos/etcd/clientv3"
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	DefaultEtcdTimeout time.Duration = 5 * time.Second
+)
+
+// TODO
+// 1. add watcher to watch the change state of pservers)
```

Comment:
You can use [gometalinter](https://github.com/alecthomas/gometalinter) to check the code styles.